lib/medtronic/processData.js

```diff
@@ -426,6 +467,84 @@ function buildBGRecords(records) {
   return postrecords;
 }
 
+function buildCGMRecords(events) {
```

Comment:
I'd suggest that @pazaan might be better suited to reviewing the intricacies here, if y'all haven't already discussed them.